packages/components/src/BulkSelect/BulkSelect.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useRef, Fragment } from 'react';
+import classnames from 'classnames';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownItemProps,
+  DropdownToggle,
+  DropdownToggleProps,
+  DropdownToggleCheckbox,
+  DropdownToggleCheckboxProps,
+  Checkbox,
+} from '@patternfly/react-core';
+import { getDefaultOUIAId } from '@patternfly/react-core/';
+import './bulk-select.scss';
+
+export type BulkSelectItem = {
+  key?: string | number;
+  title?: string;
+  props?: DropdownItemProps;
+  onClick?: (
```

Comment:
I think this one has a different type:

`oneItem.onClick(event, oneItem, key)`

Reply:
Yep, it does but we are extending it. The dropdown item only has the `event` argument but we also send the item and key.